src/Teams.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/macro'
+import createGroup from './services/createGroups'
+
+export default function Teams({ data }) {
+  const groups = createGroup(data)
+
+  return (
+    <List>
+      {groups?.map((group, index) => {
+        return (
+          <li key={index}>
```

Comment:
I think using index here might lead to bugs when group size stays the same, not sure.